tests/unit/date-helper.spec.ts

```diff
@@ -2,10 +2,10 @@ import { shortPacificDate } from '../../src/utils/date-helper'
 
 describe('Date Helper', () => {
   it('returns short pacific date shortPacificDate()', () => {
-    expect(shortPacificDate('2021-07-01T00:00:00+00:00')).toBe('June 30, 2021')
+    expect(shortPacificDate(new Date('2021-06-30 07:00:00 GMT'))).toBe('June 30, 2021')
   })
 
   it('returns RangeError when empty string is passed shortPacificDate()', () => {
-    expect(() => shortPacificDate('')).toThrowError('Invalid time value')
+    expect(shortPacificDate('')).toBe(null)
   })
```

Comment:
Please fix the unit test label above.

Also, please add a unit test for an invalid-format date (like a string that isn't a date).

Reply:
Done